homeassistant/components/rfxtrx/config_flow.py

```diff
@@ -0,0 +1,22 @@
+"""Config flow for RFXCOM RFXtrx integration."""
+import logging
+
+from homeassistant import config_entries
+
+from . import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for RFXCOM RFXtrx."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    async def async_step_import(self, import_config=None):
+        """Handle the initial step."""
+
+        await self.async_set_unique_id(DOMAIN)
+        self._abort_if_unique_id_configured(import_config)
```

Comment:
This is fine… but it does mean that updating configuration.yaml doesn't update the config entry.

Reply:
Hmm. i thought it did? that is why import_config is passed into _abort_if_unique_id_configured?

Reply:
And I even test for it and it seem to work. Sooo... what gives.